components/script/dom/windowproxy.rs

```diff
@@ -390,47 +390,49 @@ impl WindowProxy {
         target: DOMString,
         features: DOMString,
     ) -> Option<DomRoot<WindowProxy>> {
-        // Step 3.
+        // Step 4.
         let non_empty_target = match target.as_ref() {
             "" => DOMString::from("_blank"),
             _ => target,
         };
-        // Step 4
-        let features = tokenize_open_features(features);
         // Step 5
+        let features = tokenize_open_features(features);
+        // Step 7
         let noopener = parse_open_feature_boolean(&features, "noopener");
-        // Step 6, 7
+        // TODO Step 8 & 9, handle noreferrer
```

Comment:
Despite that we agreed to drop the `noreferrer` changes from step 14.3, you could still do step 8 and 9?